sources/packages/web/src/services/http/InstitutionLocation.ts

```diff
@@ -25,12 +29,16 @@ export class InstitutionLocationApi extends HttpBaseClient {
       | InstitutionLocationPrimaryContactAPIInDTO
       | InstitutionLocationAPIInDTO,
   ): Promise<void> {
-    return this.patchCall<
-      InstitutionLocationPrimaryContactAPIInDTO | InstitutionLocationAPIInDTO
-    >(
-      this.addClientRoot(`location/${locationId}`),
-      updateInstitutionLocationDto,
-    );
+    try {
+      return await this.patchCall<
+        InstitutionLocationPrimaryContactAPIInDTO | InstitutionLocationAPIInDTO
```

Comment:
Just a heads up for all devs, we do not need to leave the type explicit for this situation.
It will be typed either way.
![image](https://user-images.githubusercontent.com/61259237/204913595-d7bea803-769b-4c41-a4db-c99b7c1cbcf3.png)

IMO, in this situation, it will just make the code harder to read.
@andrepestana-aot, no need to change it, just something that I believe that we can avoid.